<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间间隔计算器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="2">
    <widget class="QPushButton" name="pushButtonCalc_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>起始日期：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>间隔天数：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>得到日期：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入秒数：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Input_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd : HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>得到日期：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Output_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QCheckBox" name="checkBox_LocalTime">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UTC时间</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QPushButton" name="pushButtonCalc">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkBox_Change">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入切换</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QDateEdit" name="dateEdit_Start">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Input">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Output">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>lineEdit_Input_2</zorder>
  <zorder>dateTimeEdit</zorder>
  <zorder>pushButtonCalc_2</zorder>
  <zorder>label</zorder>
  <zorder>dateEdit_Start</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_Input</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_Output</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_Output_2</zorder>
  <zorder>checkBox_LocalTime</zorder>
  <zorder>pushButtonCalc</zorder>
  <zorder>checkBox_Change</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
